cmd/system-probe/subcommands/run/command.go

```diff
@@ -193,6 +197,13 @@ func startSystemProbe(cliParams *cliParams, log log.Component, sysprobeconfig sy
 
 	setupInternalProfiling(sysprobeconfig, configPrefix, log)
 
+	if ddconfig.Datadog.GetBool("remote_configuration.enabled") {
+		err = rcclient.Listen("system-probe", []data.Product{data.ProductAgentConfig})
```

Comment:
Does the RC client directly talk to the backend? We don't currently do that for security reasons, but are revisiting that decision.

Reply:
The RC clients don't talk to the backend directly. The flow is:
```mermaid
graph TD;
    SP[System Probe]-->CA[Core Agent];
    CA[Core Agent]-->RCB[RC Backend];
```

(More info [here](https://datadoghq.atlassian.net/wiki/spaces/RCI/pages/2273575698/Architecture) if you want)

Reply:
The linked diagram says the call to the agent is authenticated. How is that authentication taking place? I don't see it mentioned in the RFC.